packages/react-adminer/src/components/Edit.tsx

```diff
@@ -186,9 +185,26 @@ const EditChild: React.FC<Props> = ({ entityConfig, entityName, id }) => {
 		if (state[f.name] && editable && DATE_FORMATS[f.type]) {
 			return moment().format(DATE_FORMATS[f.type]);
 		}
+
 		return null;
 	};
 
+	const sectionFields = (inputFields: PrimitiveField[], selectedSection?: string): PrimitiveField[] => {
```

Comment:
Přidej to do utilitních funkcí pls, ať to není v těle renderu. To je zbytečné, akorát nám roste. není lepší to pojemnovat "getSectionFields"?